### PR TITLE
Added ProfileFields constants to LinkedIn provider

### PR DESCRIPTION
**Pull request by xperiandri**

### CODE REVIEW

File: src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs [hunk 1]
```diff
@@ -30,5 +30,110 @@ public static class Claims
...
+        public const string EmailAddressField = "email-address";
+
+        // https://developer.linkedin.com/docs/fields/basic-profile
+        public static class ProfileFields
+        {
+            /// <summary>
+            /// A unique identifying value for the member.
+            ///This value is linked to your specific application.Any attempts to use it with a different application will result in a "404 - Invalid member id" error.
+            /// </summary>
+            public const string Id = "id";
```
Comment (martincostello) [inline src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs:43]: Could you add new lines between each new constant please? It makes the code easier to read and review.

Comment (xperiandri) [reply src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs:43]: Done
Could you set up `.editorconfig` with this rule?

Comment (martincostello) [reply src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs:43]: Sure, that's something I can look at doing some time this week or next.

File: src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs [hunk 2]
```diff
@@ -30,5 +30,130 @@ public static class Claims
             public const string Specialties = "urn:linkedin:specialties";
             public const string Summary = "urn:linkedin:summary";
         }
+
+        public const string EmailAddressField = "email-address";
```
Comment (martincostello) [inline src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs:34]: I'm probably missing some nuance, but what's the difference between this one being here and the others being in the subclass?

Comment (xperiandri) [reply src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs:34]: See https://developer.linkedin.com/docs/fields/basic-profile#
It just belongs to the other group

File: src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs (hunk 1 above)
Comment (martincostello) [reply src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationConstants.cs:43]: #296

**Review by martincostello — APPROVED** [review]

**Pull request merged**